SteamKit2/SteamKit2/Steam/WebAPI/SteamConfigurationWebAPIExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace SteamKit2
+{
+    /// <summary>
+    /// Provides helper extensions to make WebAPI interfaces from existing SteamConfiguration.
+    /// </summary>
+    public static class SteamConfigurationWebAPIExtensions
+    {
+        /// <summary>
+        /// Retrieves a dynamic handler capable of interacting with the specified interface on the Web API.
+        /// </summary>
+        /// <param name="config">The configuration to use for this Web API interface.</param>
+        /// <param name="iface">The interface to retrieve a handler for.</param>
+        /// <returns>A dynamic <see cref="WebAPI.Interface"/> object to interact with the Web API.</returns>
+        public static WebAPI.Interface GetWebAPIInterface(this SteamConfiguration config, string iface)
+        {
+            if (config == null)
```

Comment:
Is it even possible for this to be `null` if it's extension method?

Reply:
Yes. You can call an extension method on `null`, or manually call this function since it's a public static function.